app/models/issue_tta_report.rb

```diff
@@ -0,0 +1,4 @@
+class IssueTtaReport < ApplicationRecord
+  validates :issue_id, :tta_report_display_id, :tta_report_id, presence: true
+  validates :tta_report_id, numericality: {only_integer: true}
```

Comment:
i don't feel strongly one way or another -- i was just mimicking TTA's data types. 

what do you mean by "safer" in this instance?

Reply:
Safer in that a change from integers to UUID wouldn't need to result in a db migration for us (though it would still require a code change for the validation)

Reply:
I guess I would just be surprised if TTAHub changed the data type of the ActivityReport's id, but migrations are inherently riskier than changing validations. what do you think @jduss4?

Reply:
I would also be surprised by that, but weirder things have happened. A more likely possibility is way down the road when `integer` -> `bigint`

Either way, my rule of thumb is to use strings for most things that don't involve math, even if they happen to currently be numbers

Reply:
I think that using a string is fine, especially if we're using validation, so that this will match the other identifier fields.  That being said, I don't have a problem with it being an integer if we're expecting an integer.  Migrating to bigint shouldn't be a big deal in the future, if it was needed.  My official vote is slight preference towards string but I don't mind integer.